src/metrology_apis/__init__.py

```diff
@@ -107,32 +189,19 @@ def __rmul__(self, other: Self, /) -> Self: ...
     def __rtruediv__(self, other: Self, /) -> Self: ...
 
 
+Unit.__metrology_namespace__.__doc__ = (
+    HasMetrologyNamespace.__metrology_namespace__.__doc__
+)
+
+# ===================================================================
+# Quantity
+
+
 @runtime_checkable
 class Quantity[V: VT = Any, U: UT = Any, D: Dimension = Any](Protocol):
```

Comment:
now you've added DT, should Quantity and Unit use it for consistency?

Reply:
@andrewgsavage Done!